### PR TITLE
Bandit fix

### PR DESCRIPTION
**Pull request by albertkol**
I tried a few different things example verbose run: 
https://github.com/communitiesuk/funding-service-design-authenticator/actions/runs/11631722517/job/32393348190?pr=349

It's running on everything inside the project...
`-lll` makes it to throw an error only for `High`  level severity issues.

## How to test?
It's a bit annoying. But perhaps easiest is to clone [this branch](https://github.com/communitiesuk/funding-service-design-authenticator/pull/349/files).

If you want to test on local.
1. cd to authenticator dir
2. source venv
3. `pip install -r requirements-dev.txt bandit==1.7.4`
4. `bandit -v -r . -x /.venv -lll` (you have to ignore `.venv` on local)

### CODE REVIEW

File: .github/workflows/post-deploy.yml
Comment (samuelhwilliams) [inline .github/workflows/post-deploy.yml:136]: Am I reading/interpreting correctly that this step runs after the code has already been merged to main, or? That feels like the wrong time to be running this step because it'll block the pipeline.

(Aware this would clearly be a pre-existing issue 👀)

**Review by samuelhwilliams — APPROVED** [review]

File: .github/workflows/post-deploy.yml
Comment (albertkol) [reply .github/workflows/post-deploy.yml:136]: I thought the same.  

We should take a look at those workflows.

**Pull request merged**
